src/idm/index.js

```diff
@@ -0,0 +1,37 @@
+import axios from 'axios';
+import group from './group';
+import user from './user';
+import entity from './entity';
+import authentication from './authentication';
+
+
+import { errorHandler } from '../utils';
+
+const idm = (base, token) => {
+  base = `${base}`;
+  return ({
+    /**
+    * @namespace group
+    * @memberof agile.idm
+    **/
+    group: group(base,token),
+    /**
+    * @namespace user
+    * @memberof agile.idm
+    **/
+    user: user(base,token),
+    /**
+    * @namespace entity
+    * @memberof agile.idm
+    **/
+    entity: entity(base,token),
+    /**
+    * @namespace authentication
```

Comment:
It would be nice to have the token in a global state within the api, but it cannot be constant, because the token could expire. I think this is nice to have, but we can sort this out in the future, for now, I am more concerned to make sure that we update the interfaces that could affect other components and then we can refine the internals of the implementation for keeping an updated token. For now, I would recreate a new agilesdk when the token expires and improve this latter. Do you agree?